### PR TITLE
MAINT: some tweaks to spatial.procrustes (private file, html docs, style)

### PR DESCRIPTION
**Pull request by rgommers**
Also updated the release notes.

### CODE REVIEW

File: doc/release/0.16.0-notes.rst [hunk 1]
```diff
@@ -40,6 +40,9 @@ square root of the i-th updated eigenvalue of a positive symmetric rank-one
 modification to a positive diagonal matrix. See its LAPACK documentation and
 unit tests for it to get more info.
 
+The function `scipy.linalg.orthogonal_procrustes`, which computes the matrix
```
Comment (ewmoore) [inline doc/release/0.16.0-notes.rst]: This was actually in 0.15 it was #3809.

**Comment by ewmoore** [comment]
I guess I jumped the gun a little earlier.  Thanks for cleaning this up.

File: doc/release/0.16.0-notes.rst (hunk 1 above)
Comment (rgommers) [inline doc/release/0.16.0-notes.rst]: fixed, thanks

**Comment by rgommers** [comment]
@ewmoore no worries, the PR was in pretty good shape. These are all details I'd say.

**Comment by rgommers** [comment]
With keeping new files added private instead of making them look like new submodules we should try to be strict though, which is why I took the trouble to submit this PR.

File: scipy/spatial/_procrustes.py [hunk 2]
```diff
@@ -38,21 +38,20 @@ def procrustes(data1, data2):
...
-        matrix, n rows represent points in k (columns) space data1 is the
+        Matrix, n rows represent points in k (columns) space data1 is the
         reference data, after it is standardised, the data from data2 will
         be transformed to fit the pattern in data1 (must have >1 unique
         points).
-
     data2 : array_like
-        n rows of data in k space to be fit to data1.  Must be the  same
-        shape (numrows, numcols) as data1 (must have >1 unique points).
+        n rows of data in k space to be fit to `data1`.  Must be the  same
```
Comment (ElDeveloper) [inline scipy/spatial/_procrustes.py]: I think the backticks should be removed from this "data1" or backticks should be added for the other occurrences of data1 and data2 in the description of the parameters. Not a problem really, just an observation.

**Comment by ElDeveloper** [comment]
:+1:

File: scipy/spatial/_procrustes.py (hunk 2 above)
Comment (rgommers) [inline scipy/spatial/_procrustes.py]: good point, done.

**Comment by ev-br** [comment]
Thanks Ralf

**Pull request merged**